Whiskey/Tasks/CreateFile.ps1

```diff
@@ -0,0 +1,69 @@
+function New-WhiskeyFile
+{
+    [Whiskey.Task('CreateFile')]
+    [CmdletBinding()]
+    param(
+        [Whiskey.Tasks.ValidatePath(Mandatory,PathType='File',MustExist=$false,AllowAbsolute)]
+        [string[]]$Path,
+
+        [Whiskey.Context]$TaskContext,
+
+        [string]$Content,
+
+        [bool]$Force,
+
+        [bool]$Touch
+    )
+
+    Set-StrictMode -Version 'Latest'
+    Use-CallerPreference -Cmdlet $PSCmdlet -SessionState $ExecutionContext.SessionState
+
+    if (-not $TaskContext)

```

Comment:
Instead of this check, add `[Parameter(Mandatory)]` to the `$TaskContext` parameter.